www/front_src/src/Login/Logo.tsx

```diff
@@ -1,10 +1,17 @@
 import * as React from 'react';
 
+import { useAtomValue } from 'jotai/utils';
+import { equals } from 'ramda';
 import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
 
 import { makeStyles } from '@mui/styles';
 
+import { useMemoComponent } from '@centreon/ui';
+import { ThemeMode, userAtom } from '@centreon/ui-context';
+
 import logoCentreon from '../assets/centreon.png';
+import logoDark from '../assets/centreon-logo-white.svg';
```

Comment:
I think for consistency, I would choose names of what the logo represents, not the functionality.

logo-centreon-white, logo-centreon-dark, and logo-centreon-colors